analytics/event.go

```diff
@@ -0,0 +1,38 @@
+package analytics
+
+// EventType enumerates the values of events Prebid Server can receive for an ad.
+type EventType string
+
+// Possible values of  vents Prebid Server can receive for an ad.
+const (
+	Win EventType = "win"
+	Imp EventType = "imp"
+)
+
+// EventFormat enumerates the values of a Prebid Server event.
+type ResponseFormat string
+
+const (
+	// BlankEventFormat describes an event which returns an HTTP 200 with an empty body.
+	Blank ResponseFormat = "b"
+	// ImageEventFormat describes an event which returns an HTTP 200 with a PNG body.
+	Image ResponseFormat = "i"
+)
+
+// Indicates if the notification event should be handled or not
+type Analytics string
```

Comment:
Nitpick: Please rename start of the comment to match the new type name `Analytics indicates if ...`.